weatherAplication/Model/DataModels/OpenMeteo/HourlyWeatherData.swift

```diff
@@ -2,10 +2,10 @@ import UIKit
 
 struct HourlyWeatherData: Decodable {
     let time: [String]
-    let temperature: [Double]
-    let surfacePressure: [Double]
-    let weathercode: [Int]
-    let windspeed: [Double]
+    let temperature: [Double?]
+    let surfacePressure: [Double?]
+    let weathercode: [Int?]
+    let windspeed: [Double?]
```

Comment:
skoro masz codinkkeys to lepiej sie trzymac camelCasa, czyli weatherCode, windSpeed